tests/nnapi/specs/Ex/one_hot_1_ex_dynamic_nnfw.mod.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2018 The Android Open Source Project
+# Copyright (C) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# If depth or axis is from input, the output shape of one_hot cannot be decided at compilation time.
+# Therefore, its output is "dynamic tensor"
+#
+
+import dynamic_tensor
+
+model = Model()
+
+indice_shape = [1]
```

Comment:
This is the same size as a scalar. Could you change it to a larger size?

Reply:
Ok, I will.